elastic/integrations
# packages/apm: remove data streams in 8.15.0

## Proposed commit message

As of https://github.com/elastic/elasticsearch/pull/108860 (due to be released in 8.15.0), the apm-data Elasticsearch plugin is enabled by default and will take care of installing index templates and ingest pipelines for APM; the integration package no longer needs to (nor should) install its own.

With this change, the integration package is optional for APM: it is still useful for running with Fleet, but otherwise users may rely on templates/pipelines being set up by Elasticsearch.

## Checklist


## Author's Checklist

- [ ] Test installing this version of the integration package and ingesting data with APM Server

## How to test this PR locally



1. Start Elasticsearch, Kibana, and APM Server (binary)
2. Install the APM integration package
3. Ingest data through APM Server, confirm it is indexed correctly/visualised correctly in Kibana

## Related issues

https://github.com/elastic/apm-server/issues/11529
